apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c

```diff
@@ -810,6 +844,7 @@ static void utc_audio_pcm_writei_p(void)
 	char *buffer;
 	int num_read;
 	unsigned int size;
+	int retry = 10;
```

Comment:
#define PREPARE_RETRY_COUNT 10(Actually 10 is too large, I think )

Reply:
Done. 
I agree that retry count of 10 is big. But I think its ok for UTC. 
We can have a smaller count in media player.